### ISSUE TITLE
Tests duplicates with multiple files

### ISSUE THREAD

**Issue by stgogm** [nordcloud/serverless-mocha-plugin#102]
When running tests, the mocha reporter shows that the tests are run twice (or more depending on the number of files).

This is a run with two test files:

* `test/functions/test.spec.js`
* `test/functions/ads/index.spec.js`

```
$ sls invoke test --stage local --path test/functions

  Ads
  Ads
    index
    index
      ✓ should respond 204 (NoContent)
      ✓ should respond 204 (NoContent)

  Test

  Test
    ✓ should respond 204 (NoContent)
    ✓ should respond 204 (NoContent)



  2 passing (7ms)


  2 passing (5ms)
```

This is a run with three files:

* `test/functions/test.spec.js`
* `test/functions/ads/index.spec.js`
* `test/functions/ads/view.spec.js`

```
$ sls invoke test --stage local --path test/functions

  Ads
  Ads
  Ads
    index
    index
    index
      ✓ should respond 204 (NoContent)
      ✓ should respond 204 (NoContent)
      ✓ should respond 204 (NoContent)

  Ads

  Ads

  Ads
    view
    view
    view
      ✓ should respond 204 (NoContent)
      ✓ should respond 204 (NoContent)
      ✓ should respond 204 (NoContent)

  Test

  Test

  Test
    ✓ should respond 204 (NoContent)
    ✓ should respond 204 (NoContent)
    ✓ should respond 204 (NoContent)




  3 passing (8ms)


  3 passing (8ms)


  3 passing (7ms)
```

And the tests are defined as:

```js
const mochaPlugin = require('serverless-mocha-plugin');
const { expect } = mochaPlugin.chai;

let wrapped = mochaPlugin.getWrapper('test', '/service/functions/test', 'handler');

describe('Test', function() {
  this.timeout(30000);

  it('should respond 204 (NoContent)', async () => {
    const res = await wrapped.run({});

    expect(res).to.not.be.empty;
    expect(res.statusCode).to.equal(204);
    expect(res.body).to.be.empty;
  });
});
```

If I add more files, the tests are run the same amount of tests files there are.

I might be missing something...

Please let me know if you need more info.

Thanks!

**Comment by tierfire** [comment]
I am running into the same issue.

**Comment by mpuittinen** [comment]
@tierfire @stgogm Thanks for the report. Fixed in serverless-mocha-plugin 1.9.2
